plugins/labhub.py

```diff
@@ -381,3 +381,100 @@ def pr_stats(self, msg, match):
                     state=type(self).community_state(pr_count)
                  )
         yield reply
+
+    @re_botcmd(pattern=r'^migrate\s+https://github.com/([^/]+)/([^/]+)/+issues/(\d+)\s+https://github.com/([^/]+)/([^/]+)/*$',  # Ignore LineLengthBear, PyCodeStyleBear
+               # Ignore LineLengthBear, PyCodeStyleBear
+               re_cmd_name_help='migrate <complete-issue-URL-to-be-copied-from> <complete-repo-URL-to-be-copied-into>',
+               flags=re.IGNORECASE)
+    def migrate_issue(self, msg, match):
+        """
+        Migrate an issue from one repo
+        to another repo of coala
+        """
+        org = match.group(1)
+        repo_name_orig = match.group(2)
+        issue_number = match.group(3)
+        user = msg.frm.nick
+        org2 = match.group(4)
+        repo_name_final = match.group(5)
+
+        try:
+            assert org == self.GH_ORG_NAME or org == self.GL_ORG_NAME
+        except AssertionError:
+            yield 'First repository not owned by our org.'
+            return
+
+        try:
+            assert org2 == self.GH_ORG_NAME or org2 == self.GL_ORG_NAME
+        except AssertionError:
+            yield 'Second repository not owned by our org.'
+            return
+
+        if (repo_name_orig in self.REPOS) and (repo_name_final in self.REPOS):
+            pass
+        else:
+            yield 'Repository does not exist!'
+            return
+
+        if self.TEAMS[self.GH_ORG_NAME + ' maintainers'].is_member(user):
+            pass
+        else:
+            yield tenv().get_template(
+                'labhub/errors/not-maintainer.jinja2.md'
+            ).render(
+                action='migrate issues',
+                target=user,
+            )
+            return
+
+        try:
+            old_issue = self.REPOS[repo_name_orig].get_issue(int(issue_number))
+            old_labels = old_issue.labels
+
+        except RuntimeError:
+            yield 'Issue does not exist!'
+            return
+
+        if str(old_issue.state) == 'closed':
+            yield 'Issue cannot be migrated as it has been closed already.'
+            return
+        else:
+            pass
+
+        url1 = 'https://github.com/{}/{}/issues/{}'
+        new_issue_title = old_issue.title
+        new_issue_description = old_issue.description.rstrip()
+        extra_msg = '\n\nMigrated issue from '+url1.format(
```

Comment:
maybe include the author as well? `Migrate issue opened by @author from ...` so the author the issue follows new issue as well

Reply:
I was trying the same, but there seems to be some problem with the `IGitt` module I have installed on my system using `pip`, as it says author of the issue is not an attribute even though I find this method here: https://gitlab.com/gitmate/open-source/IGitt/blob/master/IGitt/GitHub/GitHubIssue.py and I am not particularly sure why... :sweat_smile: 
![screenshot from 2018-04-08 00-26-01](https://user-images.githubusercontent.com/20523845/38459262-38e3e0d2-3ac5-11e8-87bc-84513bc9b0d0.png)